library/include/rocblas_bfloat16.h

```diff
@@ -73,6 +73,11 @@ struct rocblas_bfloat16
         return u.fp32;
     }
 
+    explicit __host__ __device__ constexpr operator double() const
```

Comment:
can you change to

explicit constexpr __host__ __device__ operator double() const

This will make it consistent with the other functions that have constexpr before __host__

Reply:
modified in commit "add document for gemm_ex and correct func format"

Reply:
`rocblas_bfloat16.h` has been updated on develop. This version is outdated.

We should not convert from `rocblas_bfloat16` to `double`. The conversion from `rocblas_bfloat16` to `float` is exact, and it can be up-converted to other types from there. We do not need to list every type conversion.